images/prow-tests/Dockerfile

```diff
@@ -29,13 +24,6 @@ RUN gcloud components update
 RUN gcloud components install docker-credential-gcr
 RUN docker-credential-gcr configure-docker
 
-# Replace kubectl with a working version
-# TODO(chizhg): remove once https://github.com/knative/test-infra/issues/1858 is fixed
-ARG KUBECTL_VERSION=v1.17.4
```

Comment:
This cannot be removed yet as some old release versions are still using the old way of installing Knative serving - https://github.com/knative/serving/blob/release-0.15/vendor/knative.dev/test-infra/scripts/library.sh#L489-L499.

Could you update the comment to say `remove once test infra stops supporting releases <=0.15`?

Reply:
done